lib/shift_values.rb

```diff
@@ -0,0 +1,20 @@
+class ShiftValues
+
+  def squared(date)
+    (date.to_i)**2
+  end
+
+  def key_values(num)
+    num.chars.each_cons(2).map {|numbers| numbers.join.to_i}
+  end
+
+  def offset_values(date)
+    squared(date).to_s.chars.last(4).map {|number| number.to_i}
```

Comment:
uses helper method (squared) to return an integer
turns integer into a string, breaks it up into an array with each digit string,
grabs the last four digits, and turn them into an integer
returns an array of 4 integers